### DISCUSSION TITLE
Minecraft版本怎么变草方块了？

### DISCUSSION THREAD

**Discussion by Chormore-EDGE** [Meloong-Git/PCL#2024] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/114737601/313c351d-30d0-43a6-8738-99be6cf0d99a)

**Comment by WForst-Breeze** [comment, marked as answer]
这是 Mojang 在 23w16a 做的更改，游戏的应用图标由工作台改为草方块，若是快照则为泥土。
